src/main/java/org/snlab/unicorn/UnicornService.java

```diff
@@ -1,9 +1,8 @@
 package org.snlab.unicorn;
 
+import java.nio.file.Paths;
 import java.util.UUID;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
```

Comment:
If you are using IntelliJ, you can set the maximum import count in **Preferences -> Editor -> Code Style -> Java -> Imports" by changing **Class count to use import with '*'** and **Names count to use static import with '*'** to 999.

It will make you avoid introducing wildcard imports.